webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -129,56 +133,56 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
         id={path}
         ref={bodyRef}
       >
-        <tr>
-          <td
-            className={cx({ [styles.pinnedColumnCell]: pinnedColumn })}
-            colSpan={pinnedColumn ? 1 : nbColumns}
+        <ComparisonTablePinnedContentRow
+          pinnedColumn={pinnedColumn}
+          nbColumns={nbColumns}
+        >
+          <div className={styles.rowPath}>
+            <button className={styles.rowToggler} onClick={toggleIsShownState}>
+              <Icon icon={isShown ? ChevronDown : ChevronRight} />
+              <Tooltip
+                content={path}
+                placement="bottom-start"
+                delay={NORMAL_TOOLTIP_DELAY}
+              >
+                <span className={styles.pathText}>{path}</span>
+              </Tooltip>
+            </button>
+            <CopyButton value={path} className={styles.copyButton} />
+          </div>
+        </ComparisonTablePinnedContentRow>
+        {boundingBoxClassesArr.length > 0 && (
+          <ComparisonTablePinnedContentRow
+            pinnedColumn={pinnedColumn}
+            nbColumns={nbColumns}
           >
-            <div className={styles.rowHeader} data-testid="row-header">
-              <div className={styles.rowHeaderPath}>
-                <button
-                  className={styles.rowToggler}
-                  onClick={toggleIsShownState}
-                >
-                  <Icon icon={isShown ? ChevronDown : ChevronRight} />
-                  <Tooltip
-                    content={path}
-                    placement="bottom-start"
-                    delay={NORMAL_TOOLTIP_DELAY}
+            <div
+              data-testid="row-bounding-box-classes"
+              className={cx(styles.boundingBoxClasses, cellClasses)}
```

Comment:
I did test out making the animation of the classes collapsing a bit smoother, but decided to keep it as is:

https://github.com/iterative/vscode-dvc/assets/43496356/d1591e5c-bf92-482b-a07a-4e821f83816f